src/Jackett.Common/Indexers/DivxTotal.cs

```diff
@@ -0,0 +1,333 @@
+
+                if (result.Status != HttpStatusCode.OK)
+                    throw new ExceptionWithConfigData(result.Content, configData);
+
+                try
+                {
+                    var searchResultParser = new HtmlParser();
+                    var doc = searchResultParser.ParseDocument(result.Content);
+
+                    var table = doc.QuerySelector("table.table");
+                    var rows = table.QuerySelectorAll("tr");
+                    isLastPage = rows.Length -1 < MAX_RESULTS_PER_PAGE; // rows includes the header
+                    var isHeader = true;
+                    foreach (var row in rows)
+                    {
+                        if (isHeader) {
+                            isHeader = false;
+                            continue;
+                        }
+
+                        await ParseRelease(releases, row, queryStr, query.Categories, matchWords);
+                    }
+                }
+                catch (Exception ex)
+                {
+                    OnParseError(result.Content, ex);
+                }
+
+                page++; // update page number
+
+            } while (!isLastPage && page <= MAX_SEARCH_PAGE_LIMIT);
+
+            return releases;
+        }
+
+        public override async Task<byte[]> Download(Uri link)
+        {
+            // for tv series we already have the link
+            var downloadUrl = link.ToString();
+            // for other categories we have to do another step
+            if (!downloadUrl.EndsWith(".torrent"))
+            {
+                var result = await RequestStringWithCookies(downloadUrl);
+
+                if (result.Status != HttpStatusCode.OK)
+                    throw new ExceptionWithConfigData(result.Content, configData);
+
+                var searchResultParser = new HtmlParser();
+                var doc = searchResultParser.ParseDocument(result.Content);
+
+                var onclick = doc.QuerySelector("a[onclick*=\"/download/torrent.php\"]")
+                    .GetAttribute("onclick");
+                downloadUrl = OnclickToDownloadLink(onclick);
+            }
+
+            var content = await base.Download(new Uri(downloadUrl));
+            return content;
+        }
+
+        private async Task ParseRelease(List<ReleaseInfo> releases, IElement row, string queryStr, int[] queryCats,
+            bool matchWords)
+        {
+            var anchor = row.QuerySelector("a");
+            var commentsLink = anchor.GetAttribute("href");
+            var title = anchor.TextContent.Trim();
+            var cat = commentsLink.Split('/')[3];
+            var categories = MapTrackerCatToNewznab(cat);
+            var publishStr = row.QuerySelectorAll("td")[2].TextContent.Trim();
+            var publishDate = TryToParseDate(publishStr, DateTime.Now);
+            var sizeStr = row.QuerySelectorAll("td")[3].TextContent.Trim();
+            var size = TryToParseSize(sizeStr, 0);
```

Comment:
the default size when none is available is usually set to 500MB
this is because some torznab apps reject/(refuse to download)  0B files.

Reply:
done